src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/NodeLabelMappingToLiteralsImpl.java

```diff
@@ -0,0 +1,24 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+
+public class NodeLabelMappingToLiteralsImpl implements NodeLabelMapping{
+
+    @Override
+    public Node map(final String label) {
+        return NodeFactory.createLiteral(label);
+    }
+
+    @Override
+    public String unmap(Node node) {
+        if (!node.isLiteral())
```

Comment:
Additionally check that (if it is a literal) the datatype is of this literal xsd:string.